From d0ace108e5f8ec75dbefca8120827e8f3634180c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Tue, 10 Sep 2019 04:44:10 +0200
Subject: [PATCH 037/152] calendar-server: Disable deprecated e-d-s API

We aren't using any deprecated evolution-data-server API, so we can
turn it off; this avoids compiler warnings for glib deprecations
used by those functions, which makes it harder to spot warnings for
our own code base.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/709
---
 src/calendar-server/calendar-sources.c            | 1 +
 src/calendar-server/gnome-shell-calendar-server.c | 1 +
 2 files changed, 2 insertions(+)

diff --git a/src/calendar-server/calendar-sources.c b/src/calendar-server/calendar-sources.c
index 38b765ecc..aa3fa64f9 100644
--- a/src/calendar-server/calendar-sources.c
+++ b/src/calendar-server/calendar-sources.c
@@ -28,6 +28,7 @@
 #include <libintl.h>
 #include <string.h>
 #define HANDLE_LIBICAL_MEMORY
+#define EDS_DISABLE_DEPRECATED
 #include <libecal/libecal.h>
 
 #undef CALENDAR_ENABLE_DEBUG
diff --git a/src/calendar-server/gnome-shell-calendar-server.c b/src/calendar-server/gnome-shell-calendar-server.c
index 2da4d113a..4eaaab4ac 100644
--- a/src/calendar-server/gnome-shell-calendar-server.c
+++ b/src/calendar-server/gnome-shell-calendar-server.c
@@ -34,6 +34,7 @@
 #include <gio/gio.h>
 
 #define HANDLE_LIBICAL_MEMORY
+#define EDS_DISABLE_DEPRECATED
 #include <libecal/libecal.h>
 
 #include "calendar-sources.h"
-- 
2.23.0

